Makefile

```diff
@@ -86,13 +86,6 @@ kustomize: mise yq ## Download kustomize locally if necessary.
 	@$(MAKE) mise-plugin-install DEP=kustomize
 	@$(MAKE) mise-install DEP_VER=kustomize@$(KUSTOMIZE_VERSION)
 
-CLIENT_GEN_VERSION = $(shell $(YQ) -ojson -r '.kube-code-generator' < $(TOOLS_VERSIONS_FILE))
-CLIENT_GEN = $(PROJECT_DIR)/bin/installs/kube-code-generator/$(CLIENT_GEN_VERSION)/bin/client-gen
-.PHONY: client-gen
-client-gen: mise yq ## Download client-gen locally if necessary.
```

Comment:
We still have this defined as dependency in 

https://github.com/Kong/kubernetes-ingress-controller/blob/7580c5434444c2498f48aa47c61832afdc423162/Makefile#L51